packages/puppeteer-core/src/common/WaitTask.ts

```diff
@@ -231,6 +231,16 @@ export class WaitTask<T = unknown> {
         return;
       }
 
+      // Errors coming from WebDriver BiDi. TODO: Adjust messages after
+      // https://github.com/w3c/webdriver-bidi/issues/540 is resolved.
+      if (
+        error.message.includes(
```

Comment:
There is the same message a few line above with a comment, pointing to the same issue.  
Let's close this PR.